src/components/Nav.js

```diff
@@ -51,6 +51,34 @@ const Nav = () => {
           </Box>
         )}
         <Search open={searchOpen} setOpen={(value) => setSearchOpen(value)} />
+        <Button
+          tip={
+            window.location.href.indexOf('hpe') > -1
```

Comment:
Seems like this should come from the theme context or at least the `themeName` stored in App.js. Since it's stateful in the App that means it may not actually always be in the `window.location`. It's also possible for it to be something other than `hpe`, `grommet` or undefined (like aruba, dark, dhc, hp...). 

And picking out a query param should instead do this by actually looking for the `theme` query parameter. App.js shows how to pick out query parameters.

I'm surprised we haven't tweaked this to use the react router. Then we could use the `useParams()` hook :)